tests/handlers/test_e2e_keys.py

```diff
@@ -145,3 +147,66 @@ def test_claim_one_time_key(self):
                 "one_time_keys": {local_user: {device_id: {"alg1:k1": "key1"}}},
             },
         )
+
+    @defer.inlineCallbacks
+    def test_replace_master_key(self):
```

Comment:
as a general note for future reference, we're moving away from this style of test towards ones which (a) inherit from `HomeserverTestCase` to instantiate a Homeserver with a mock reactor and (b) exercise the REST API rather than the mid-layer of the handlers.